### ISSUE TITLE
RDF and OWL annotations [suggestion]

### ISSUE THREAD

**Issue by charbull** [Complexible/pinto#22]
Hello, 

Is there any will to add OWL annotations? with OWL API?

Thank you,

**Comment by mhgrove** [comment]
No. But it would probably be straightforward to create new java annotations that could be used to provide the annotation information that would then be used during serialization
